services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -311,6 +311,17 @@ public Builder select(Select select) {
             return this;
         }
 
+        /**
+         * Determines the attributes to be returned in the result. See {@link Select} for examples and constraints.
+         * If not found, returns {@link Select#UNKNOWN_TO_SDK_VERSION}.
+         * @param select the selection criteria as a string
+         * @return a builder of this type
+         */
```

Comment:
nit: this API doc seems to be copied from a getter, not setter? Can we update it